src/Microsoft.Health.SqlServer/Features/Schema/Messages/Get/GetCompatibilityVersionResponse.cs

```diff
@@ -19,3 +21,5 @@ public GetCompatibilityVersionResponse(CompatibleVersions versions)
 
     public CompatibleVersions CompatibleVersions { get; }
 }
+
+#pragma warning restore CA1716 // Identifiers should not match keywords
```

Comment:
Is this needed? i.e. is it just file scoped anyway?

Reply:
Oh, I suppose that's true!